examples/ola-rdm.cpp

```diff
@@ -212,14 +213,20 @@ void DisplayHelpAndExit(const options &opts) {
  * Dump the list of known pids
  */
 void DisplayPIDsAndExit(uint16_t manufacturer_id,
-                        const PidStoreHelper &pid_helper) {
-  vector<string> pid_names;
-  pid_helper.SupportedPids(manufacturer_id, &pid_names);
-  sort(pid_names.begin(), pid_names.end());
-
-  vector<string>::const_iterator iter = pid_names.begin();
-  for (; iter != pid_names.end(); ++iter) {
-    cout << *iter << endl;
+                        const PidStoreHelper &pid_helper,
+                        bool set_mode) {
+  vector<const PidDescriptor *> pids;
+  pid_helper.SupportedPids(manufacturer_id, &pids);
+  std::sort(pids.begin(), pids.end(), &PidDescriptor::CompareByName);
```

Comment:
If you're using std::sort, you can just go for sort here. If it's only used once, I generally drop the using and keep the namespace on it.

Reply:
I usually prefer to leave the namespace in across the board (probably why I missed removing the `using` before), and it's what [Google likes to do](https://google.github.io/styleguide/cppguide.html#Namespaces).

Reply:
The Google stuff looks like its about using a namespace, rather than an class in a namespace as we're doing here. Certainly cpplint doesn't complain about this setup.